virttest/utils_test/libvirt.py

```diff
@@ -1499,6 +1499,31 @@ def check_state():
         return utils_misc.wait_for(check_state, timeout)
 
 
+def check_status_output(status, output='',
+                        expected_fails=[],
+                        skip_if=[],
+                        any_error=False,
+                        expected_match=[]):
```

Comment:
The default values should be immutable, please update to None.

Reply:
Done :)